openedx/frontend-app-discussions
# fix [BB5598][TNL-9626] update landing view

**Description**
Update Landing view tab logic based on user threads.
**Additional Information**
- https://tasks.opencraft.com/browse/BB-5598
- https://openedx.atlassian.net/browse/TNL-9626
** Testing**
- Log in with a user who have already some posts -> you should get redirected to the `My posts` tab.
- Log in with a user who doesn't have any post -> you should get redirected to the `All posts` tab.

** Check recording **
https://www.loom.com/share/bdb5edcad69d43098bc17d083b0bc001
